app/services/open_library_service.rb

```diff
@@ -0,0 +1,34 @@
+class OpenLibraryService
+  include HTTParty
+
+  base_uri Rails.application.credentials.open_library_uri
+  default_params format: 'json', jscmd: 'data'
+  format :json
+
+  def initialize; end
```

Comment:
You imagine if I suggest you to use `initialize` is not to simply add an empty method, is it?